Drivers/seds/PioHwas/example/work/hwas/implem/default/C/src/hwas.c

```diff
@@ -0,0 +1,334 @@
+/* Body file for function HWAS
```

Comment:
This header could be removed.... We will need to clean-up Hwas anyway in its original repository, but that is a task for later.

Reply:
Ok, you mean that I should remove this whole boilerplate and replace it with our license text?

Reply:
Yes

Reply:
Please just remove the boilerplate

Reply:
Fixed. Boilerplate deleted.